src/tools/perf/perftest.c

```diff
@@ -320,6 +320,7 @@ static ucs_status_t setup_sock_rte_p2p(struct perftest_context *ctx)
     struct sockaddr_in inaddr;
     struct hostent *he;
     ucs_status_t status;
+    int optval = 1;
```

Comment:
initialized vars should be before un-initialized

Reply:
it's original code

Reply:
ok